src/main/java/io/axoniq/axonserver/connector/admin/impl/AdminChannelImpl.java

```diff
@@ -91,6 +127,170 @@ private EventProcessorIdentifier eventProcessorId(String eventProcessorName, Str
                                        .build();
     }
 
+    @Override
+    public CompletableFuture<Void> createOrUpdateUser(CreateOrUpdateUserRequest request)  {
+        FutureStreamObserver<Empty> responseObserver = new FutureStreamObserver<>(null);
+        userServiceStub.createOrUpdateUser(request, responseObserver);
+        return responseObserver.thenAccept(empty -> {
+        });
+    }
+
+    @Override
+    public CompletableFuture<List<UserOverview>> getAllUsers() {
+        FutureListStreamObserver<UserOverview> responseObserver = new FutureListStreamObserver<>();
+        userServiceStub.getUsers(Empty.newBuilder().build(),responseObserver);
```

Comment:
Indentation is off around the ,
Maybe there's more of this in the PR, so I'd suggest to have Intellij use the style guide on this PR.